bin/test.sh

```diff
@@ -31,7 +31,7 @@ echo "Checking redis-server responds to ping command..."
 # Redis <4 does not support connection URLs so REDIS_URL has to be parsed:
 # https://stackoverflow.com/questions/38271281/can-i-use-redis-cli-with-a-connection-url
 REDIS_CONNECTION_ARGS="\$(echo \${REDIS_URL} | sed 's_redis://h:\(.*\)@\(.*\):\(.*\)/_-h \2 -p \3 -a \1_')"
-TEST_COMMAND="source .profile.d/redis.sh && redis-cli ${REDIS_CONNECTION_ARGS} ping"
+TEST_COMMAND="source .profile.d/redis.sh && sleep 1 && redis-cli ${REDIS_CONNECTION_ARGS} ping"
```

Comment:
`sleep`s always feel a bit hack to my. I wonder if the `-r -i` options to `redis-cli` might be better?

Reply:
I like that idea too @bensymonds. Something like `-r 5 -i 200`

Reply:
@jabrown85 The problem I see with those values is if something has gone catastrophically wrong and redis is not going to be ready _at all_, the build will take ~1000 seconds (5 * 200) to give up trying, which is a long time to wait for a CI build to finish. Given @edmorley had gone with `sleep 1` I'm assuming it's only a really short delay before it's ready, so I went with a more optimistic `-r 5 -i 2`. What do you think?

Reply:
Sorry, yeah I thought it was in milliseconds.

Reply:
Ah ok, haha. I thought you were just an extremely patient person.